static/docker-compose.yml

```
@@ -224,6 +224,41 @@ services:
     entrypoint: npm run
     command: "start:notification-sender"
 
+  event-orchestrator:
+    image: "plextrac/plextracapi:${UPGRADE_STRATEGY:-stable}"
+    deploy:
+      replicas: 1
+    depends_on:
+      - postgres
+      - redis
+    restart: always
+    ports:
+      - 4354:4354
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    volumes:
+      - uploads:/usr/src/plextrac-api/uploads:rw
+    environment:
+      COUCHBASE_URL: ${COUCHBASE_URL:-http://plextracdb}
+      CB_API_USER: ${CB_API_USER:?err}
+      CB_API_PASS: ${CB_API_PASS:?err}
+      CLIENT_DOMAIN_NAME: ${CLIENT_DOMAIN_NAME:-}
```

```suggestion
      CLIENT_DOMAIN_NAME: ${CLIENT_DOMAIN_NAME:?err}
```
To match the other spots here, and because the service [marks it as required](https://github.com/PlexTrac/product-core-backend/blob/main/apps/event-orchestrator/src/envSchema.ts#L12).